test/e2e/framework/kcp.go

```diff
@@ -86,7 +86,9 @@ func NewWorkspace(t *testing.T, config *rest.Config, options ...ClusterWorkspace
 	ctx, cancelFunc := context.WithCancel(context.Background())
 	t.Cleanup(cancelFunc)
 
-	mapper, err := apiutil.NewDynamicRESTMapper(config)
+    httpClient, err := rest.HTTPClientFor(config)
+    require.NoError(t, err)
```

Comment:
indention?

Reply:
This happens when you rewrite your editor configuration and forget to bring back auto-formatting of files 🤦🏻 . Fixed now.